commit ca503426eced5d3024efd4bfc1343fa8af892a1c
Author: Eugene Molotov <molotov@it-projects.info>
Date:   Wed Jun 18 14:15:02 2025 +0500

    Подготовили базовый docker-compose для запуска скриптов парсинга

diff --git a/docker-compose.yml b/docker-compose.yml
new file mode 100644
index 0000000..fd8c66e
--- /dev/null
+++ b/docker-compose.yml
@@ -0,0 +1,17 @@
+services:
+  standalone-chromium:
+    image: selenium/standalone-chromium:137.0
+    shm_size: "2g"
+    ports:
+      - "127.0.0.1:7900:7900"
+
+  apps:
+    depends_on:
+      - standalone-chromium
+    build:
+      context: ./docker
+      args:
+        UID: 501
+        GID: 501
+    volumes:
+      - .:/mnt/parsers
diff --git a/docker/Dockerfile b/docker/Dockerfile
new file mode 100644
index 0000000..eb41f9e
--- /dev/null
+++ b/docker/Dockerfile
@@ -0,0 +1,15 @@
+FROM registry.altlinux.org/alt/alt:p11
+USER root
+RUN apt-get update && apt-get install -y \
+    python3-module-requests \
+    python3-module-selenium
+
+ARG UID=501
+ARG GID=501
+
+RUN groupadd -g $GID -o parser
+RUN useradd -o -u $UID -g $GID -m -s /bin/bash parser
+
+USER parser
+
+WORKDIR /mnt/parsers
